public/app/features/panel/components/VizTypePicker/VisualizationSuggestions.tsx

```diff
@@ -28,19 +28,23 @@ export interface Props {
   panel?: PanelModel;
 }
 
+const useSuggestions = (data: PanelData | undefined) => {
+  const [hasFetched, setHasFetched] = useState(false);
+  const { value, loading, error, retry } = useAsyncRetry(async () => {
+    await new Promise((resolve) => setTimeout(resolve, hasFetched ? 75 : 0));
```

Comment:
this is to give the user feedback once they retry, because it's possible the retry will immediately fail.